### ISSUE TITLE
ModuleNotFoundError appears randomly, refreshing fixes

### ISSUE THREAD

**Issue by johnhamm** [CompuIves/codesandbox-client#1058]
# 🐛 bug report

## Description of the problem

Autorefresh will (of course) cause errors to appear in the view window when the source code is invalid as I'm typing.  However, I get this error that appears, and once it appears, the only way to get rid of it is to do a refresh.  I'm not sure why the error is happening, as I have all dependent modules loaded in my sandbox.

```
ModuleNotFoundError
Could not find module in path: '@angular/platform-browser/animations' relative to '/src/app/app.module.ts'
```

![2018-08-26 17_35_55-kendo - codesandbox](https://user-images.githubusercontent.com/3343496/44633993-302d5800-a961-11e8-84f2-5436e83b789e.png)

![2018-08-26 17_36_54-kendo - codesandbox](https://user-images.githubusercontent.com/3343496/44634004-4804dc00-a961-11e8-87de-1e2ba68634bd.png)

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | 
| Browser | Version 68.0.3440.106 (Official Build) (64-bit)
| Operating System |Windows 10

**Comment by CompuIves** [comment]
Hmmm, it seems like our dynamic downloading of files might be doing something random here. Could you share your dependencies from `package.json`? Then I'll try to reproduce the problem!

**Comment by johnhamm** [comment]
Yes - I have noticed that it has happened on two modules whose names are not the same as the package name:  @angular/animations/browser (package=@angular/animations), and unfortunately I don't remember the second.  I just remember wondering if that was causing the bug since the names don't match the packages.

Here's my package.json:
`{
  "name": "ui-prototype",
  "version": "0.0.0",
  "license": "MIT",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build --prod",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "6.1.4",
    "@angular/common": "6.1.4",
    "@angular/compiler": "6.1.4",
    "@angular/core": "6.1.4",
    "@angular/forms": "6.1.4",
    "@angular/http": "6.1.4",
    "@angular/platform-browser": "6.1.4",
    "@angular/platform-browser-dynamic": "6.1.4",
    "@angular/router": "6.1.4",
    "@progress/kendo-angular-buttons": "4.1.3",
    "@progress/kendo-angular-dateinputs": "3.4.4",
    "@progress/kendo-angular-dropdowns": "3.0.3",
    "@progress/kendo-angular-excel-export": "2.1.1",
    "@progress/kendo-angular-grid": "3.6.1",
    "@progress/kendo-angular-inputs": "latest",
    "@progress/kendo-angular-intl": "1.5.0",
    "@progress/kendo-angular-l10n": "1.2.0",
    "@progress/kendo-angular-layout": "3.1.2",
    "@progress/kendo-angular-popup": "2.4.1",
    "@progress/kendo-angular-treeview": "2.3.0",
    "@progress/kendo-angular-upload": "4.1.4",
    "@progress/kendo-data-query": "1.4.1",
    "@progress/kendo-drawing": "1.5.7",
    "core-js": "2.5.7",
    "mobx": "5.0.5",
    "mobx-angular": "3.0.1",
    "mocker-data-generator": "2.6.4",
    "rxjs": "6.2.2",
    "rxjs-compat": "6.2.2",
    "zone.js": "0.8.26"
  },
  "devDependencies": {
    "@angular/cli": "1.6.6",
    "@angular/compiler-cli": "^5.2.0",
    "@angular/language-service": "^5.2.0",
    "@types/core-js": "0.9.46",
    "@types/jasmine": "~2.8.3",
    "@types/jasminewd2": "~2.0.2",
    "@types/node": "~6.0.60",
    "codelyzer": "^4.0.1",
    "jasmine-core": "~2.8.0",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~2.0.0",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "^1.2.1",
    "karma-jasmine": "~1.1.0",
    "karma-jasmine-html-reporter": "^0.2.2",
    "protractor": "~5.1.2",
    "ts-node": "~4.1.0",
    "tslint": "~5.9.1",
    "typescript": "~2.5.3"
  },
  "keywords": [],
  "description": ""
}`

I can try to make a duplicatable sandbox if the package.json isn't enough.  Thanks for a great product, and I think after a few more iterations Codesandbox will be more stable than stackblitz.  It really is a great piece of software!

**Comment by johnhamm** [comment]
I have setup a sandbox that duplicates this issue:
###  Link to sandbox: [https://codesandbox.io/s/jljyv26o23](https://codesandbox.io/s/jljyv26o23)

To see the error, edit the index.html and save.

**Comment by johnhamm** [comment]
In the IDE, I get "cannot found module" errors for two modules, both of which do not match the package name.  Not sure if this is related to the bug:

![2018-08-30 23_38_05-highbeam ui prototype - codesandbox](https://user-images.githubusercontent.com/3343496/44891555-01d7b180-acae-11e8-9e61-bd0a65fb8bf4.png)

**Comment by CompuIves** [comment]
Heyo! I found the problem after looong debugging 😄. I'm waiting for the PR build to finish so I can do some more testing, if all is green I will deploy it to production!

**Comment by Slapbox** [comment]
Still seeing this problem in this sandbox https://codesandbox.io/s/redux-saga-issue-template-forked-pxbwnc?file=/src/sagas/index.js
